contracts/marketplace/src/testing/setup/setup_accounts.rs

```diff
@@ -1,10 +1,61 @@
 use crate::error::ContractError;
-use crate::testing::setup::constants::{CREATION_FEE, SECOND_BIDDER_INITIAL_BALANCE};
 use cosmwasm_std::{coins, Addr, Coin};
-use cw_multi_test::{BankSudo, SudoMsg as CwSudoMsg};
+use cw_multi_test::SudoMsg as CwSudoMsg;
+use cw_multi_test::{BankSudo, SudoMsg};
 use sg_multi_test::StargazeApp;
 use sg_std::NATIVE_DENOM;
 
+pub const INITIAL_BALANCE: u128 = 5000000000;
```

Comment:
use `_` to be consinstent with the other constants

Reply:
Done